ert3_examples/polynomial/experiments/uncertainty/ensemble.yml

```diff
@@ -0,0 +1,18 @@
+size: 10
+
+input:
+  -
+    source: stochastic.coefficients
+    record: coefficients
+  -
+    source: stochastic.x_normals
+    record: x_uncertainties
+
+output:
+  -
+    record: polynomial_output
+
+forward_model:
+  driver: local
+  stage: evaluate_x_uncertainty_polynomial
+  
```

Comment:
The end of the file should be just a new line, no white space

Reply:
I don't know why the comment above appeared in such a strange manner 🤔